h/services/user_delete.py

```diff
@@ -87,8 +116,6 @@ def purge_deleted_users(self, limit=1000):
                 completed_jobs.append(job)
                 continue
 
-            log.info("Purging user: %s", user.userid)
```

Comment:
This log message was unnecessary (you can see from all the other log messages which user is being purged)